testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -767,6 +767,38 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I use spacewalk-common-channel to add all "debian-12" channels with arch "amd64-deb"
     And I wait until all synchronized channels for "debian-12" have finished
 
+@cloud
+@proxy
+  Scenario: Add Manager Proxy 5.0 Public Cloud channels
+    When I add "sle-module-public-cloud15-sp6-pool-x86_64-proxy-5.0" channel
+    And I wait until the channel "sle-module-public-cloud15-sp6-pool-x86_64-proxy-5.0" has been synced
+    And I add "sle-module-public-cloud15-sp6-updates-x86_64-proxy-5.0" channel
```

Comment:
Are those channel names verified and correct? Just asking :wink:

Reply:
I don't remember now but I think not verified, just copy pasted